src/test-run/commands/assertion.d.ts

```diff
@@ -14,3 +14,35 @@ export default class AssertionCommand extends CommandBase {
     public options: AssertionOptions;
     public static NOT_REPORTED_PROPERTIES: string[];
 }
+
+export class Eql extends AssertionCommand {}
```

Comment:
I propose to use the long class name (`EqlAssertionCommand`) because the current class name (`Eql`) looks like enum.